docs/openstack/edpm_adoption.md

```diff
@@ -160,6 +151,8 @@ done
       - install-os
       - configure-os
       - run-os
+      - libvirt
```

Comment:
nova shoudl only be deployed on the edpm nodes after the console plane adoption is completed for nova so this is probably out of order

Reply:
it is not, EDPM adoption is executed after the Nova CP adoption is done